src/Groups/types.jl

```diff
@@ -267,6 +267,8 @@ might also be needed.
 
 elem_type(::Type{T}) where T <: GAPGroup = BasicGAPGroupElem{T}
```

Comment:
Off-topic: `elem_type` is meant to be used on objects, not types (well, it can also be used on types, but that's not the primary reason for its existence). See also https://nemocas.github.io/AbstractAlgebra.jl/latest/perm/#AbstractAlgebra.Generic.elem_type-Tuple{AbstractAlgebra.SymmetricGroup}

So the above should in fact be:
```suggestion
elem_type(::Type{T}) where T <: GAPGroup = BasicGAPGroupElem{T}
elem_type(::T) where T <: GAPGroup = BasicGAPGroupElem{T}
```

Reply:
Does the same apply for `ring_elem_type` and `mat_elem_type`?

Reply:
I don't know about `ring_elem_type` and `mat_elem_type` -- as far as I know, these are by you, and nothing currently uses them? So I think we can just drop those?